include/rosconsole_bridge/bridge.h

```diff
@@ -46,6 +46,7 @@ class OutputHandlerROS : public console_bridge::OutputHandler
 {
 public:
   OutputHandlerROS(void);
+  ~OutputHandlerROS();
```

Comment:
The base class has a virtual destructor so this should be ABI compatible.

@wjwwood Is my assumption correct?